### ISSUE TITLE
Add entry about E2EE of calls in FAQ

### ISSUE THREAD

**Issue by nahoj** [matrix-org/matrix.org#614]
I was surprised not to find on the Web whether Riot calls were E2EE and to have to ask about it on #riot-web. I suggest adding an entry such as the following to the [end-to-end encryption section of the FAQ](https://about.riot.im/help#end-to-end-encryption), assuming it is correct:

**Are Riot calls end-to-end encrypted?**

Audio and video calls are end-to-end encrypted in one-to-one chats with end-to-end encryption enabled (the number of devices doesn't matter).

End-to-end encryption of calls is not supported in group rooms.

**Comment by aaronraimist** [comment]
There is a FAq entry on this but maybe it could be reworded to make it easier to find https://matrix.org/faq/#are-voip-calls-encrypted%3F

A question really should be added to the Riot FAQ though https://about.riot.im/help

**Comment by nahoj** [comment]
> An entry on this should definitely be added to the Riot FAQ though https://about.riot.im/help

Yes, this is what I meant. Issue opened [here](https://github.com/vector-im/vector.im/issues/57) as you suggested. Thank you.

**Comment by jgoerzen** [comment]
The FAQ entry on matrix.org is hard to parse.  It doesn't answer questions such as:

- Is the video encrypted end-to-end or only to the homeserver?
- What are the implications of encrypted (or not) signaling events?
- What is different if Jitsi is used?

I do not know the answer to these myself.

**Comment by cafkafk** [comment]
I'll just bump this, surely someone must know how to answer this or it has been addressed. Else, I'll look to find out.

**Comment by 0lhi** [comment]
Wanna bump also. PrivacyGuides says one-on-one calls are E2E now, but I can't find any confirmation for this?

**Comment by MTRNord** [comment]
Webrtc is always encrypted afaik due to the protocol specification of webrtc.

Matrix signaling for the call can be encrypted. It depends on the fact if a room is encrypted or not since these are events within the matrix room. And encryption in matrix is a room global setting.

**Comment by thibaultamartin** [comment]
## Are 1:1 calls E2EE?

When you place a 1:1 call, two things happen:

- There's _signalling_ being done over Matrix. This means that the Matrix client exchange [`m.call.invite`](https://spec.matrix.org/latest/client-server-api/#mcallinvite), [`m.call.answer`](https://spec.matrix.org/latest/client-server-api/#mcallanswer) and [`m.call.candidates`](https://spec.matrix.org/latest/client-server-api/#mcallcandidates) events. The purpose of these events is to establish a WebRTC between the two clients that want to have a call. If the people who want to place a 1:1 call are in an encrypted room (like DMs have been by default for a while), those events are going to be end to end encrypted as well. You can think of those events as technical messages that are not displayed to the user, but that the client needs to know where to place the call.
- The call itself is a WebRTC stream. In some cases a TURN server is needed to relay the stream between the two peers (mostly happens for network reasons, when it's difficult to connect the two peers directly). Whether a TURN server is needed or not, the stream is E2EE between the two clients. Nobody, not even the TURN server can decrypt the stream.

You can find interesting diagrams of how it works in the context of the Android client of Element [right here](https://raw.githubusercontent.com/vector-im/element-android/develop/docs/voip_signaling.md) too. I hope this answers your question!

## Will it make to the FAQ?

We're almost done rewriting the matrix.org website. You can see a preview at https://matrix.turbo.fish, and as you'll notice there is not FAQ.

I'll have to think of a place for this information on the new website, because I really want to get rid of the FAQ. They're always outdated, the information is always a duplicate of some other place, and if you have to resort to using a FAQ it basically means that the UX/journey on your website wasn't well thought through.

**Comment by thibaultamartin** [comment]
This should not be part of the FAQ that is now obsolete, but it should probably appear somewhere in the docs

**Comment by thibaultamartin** [comment]
I have a feeling that a mention to E2EE group calls should appear early on in the documentation, e.g. https://matrix.org/docs/chat_basics/matrix-for-im/#what-is-it

Maybe the section "Matrix Concepts" would be a better fit to get details about how encrypted calls work
